examples/gnn_link.py

```diff
@@ -210,7 +210,7 @@ def test(src_loader: NeighborLoader, dst_loader: NeighborLoader) -> np.ndarray:
             f"Val metrics: {val_metrics}"
         )
 
-        if val_metrics[tune_metric] > best_val_metric:
+        if val_metrics[tune_metric] >= best_val_metric:
```

Comment:
Why is this necessary?

Reply:
If `val_metrics[tune_metric]` is never better than `0` then in L221 `model.load_state_dict(state_dict)`, `state_dict is None`, which will result in an error